meta_test.go

```diff
@@ -844,34 +629,141 @@ func TestSymmetry_json_object(t *testing.T) {
 			expectJsonEqualNonJson: true,
 		},
 	} {
-		t.Run(tc.name, func(t *testing.T) {
+		s.Run(tc.name, func() {
+			Require := s.Require()
+			Require.NoError(s.CopyMockFile(testFile))
+
 			// Get the non-json value from mock
-			stdout := new(bytes.Buffer)
-			require.NoError(t, getMeta(tc.key, mockDir, testFile, stdout, false, nil))
-			nonJsonValue := stdout.String()
+			nonJsonValue, err := nonJsonMetaSpec.Get(tc.key)
+			Require.NoError(err)
 
 			// Get the json value from mock
-			stdout = new(bytes.Buffer)
-			require.NoError(t, getMeta(tc.key, mockDir, testFile, stdout, true, nil))
-			jsonValue := stdout.String()
+			jsonValue, err := s.MetaSpec.Get(tc.key)
+			Require.NoError(err)
 
+			Assert := s.Assert()
 			// Compare starting condition
 			if tc.expectJsonEqualNonJson {
-				assert.Equal(t, jsonValue, nonJsonValue)
+				Assert.Equal(jsonValue, nonJsonValue)
 			} else {
-				assert.NotEqual(t, jsonValue, nonJsonValue)
+				Assert.NotEqual(jsonValue, nonJsonValue)
 			}
 
 			// Reset the output for writing
-			require.NoError(t, setupDir(testDir, testFile))
-			require.NoError(t, os.Remove(testFilePath))
+			_, err = s.MetaSpec.SetupDir()
+			Require.NoError(err)
+			Require.NoError(os.Remove(testFilePath))
 
 			// Set and get the jsonValue to/from writable file with jsonValue true
-			require.NoError(t, setMeta(tc.key, jsonValue, testDir, testFile, true))
-			stdout = new(bytes.Buffer)
-			require.NoError(t, getMeta(tc.key, testDir, testFile, stdout, true, nil))
-			newJsonValue := stdout.String()
-			assert.Equal(t, jsonValue, newJsonValue)
+			Require.NoError(s.MetaSpec.Set(tc.key, jsonValue))
+			newJsonValue, err := s.MetaSpec.Get(tc.key)
```

Comment:
`statuccheck`: `meta_test.go:659:18: this value of err is never used (SA4006)` - add checking the error

Reply:
Done